### DISCUSSION TITLE
Wondering targeted release date for 1.5.0

### DISCUSSION THREAD

**Discussion by rverma-dev** [aws-samples/aws-secure-environment-accelerator#805] [Q&A] [answered]
It seems 1.5.0 is going to bring a lot of features and thus making it a very awaited update :). Looking forward to get it soon.
I hope we can cutout a branch like 1.5.0-draft for the WIP code so that we can start atleast cherrypicking features in our forks.

**Comment by Brian969** [comment, marked as answer]
Doing what we can to accelerator access to the coming release.  We have started the internal release process as we finish up Control Tower integration and a script to update the config file to include the latest changes for existing customers.  Also investigating options for beta release to a few select customers.

**Comment by Brian969** [comment]
We are entering final rounds of testing, hoping to push out v1.5.0 middle of next week.

**Reply by Brian969** [reply]
Shipped.
